### PR TITLE
portfolio-Admissions trials code review

### PR DESCRIPTION
**Pull request by OyePriscilla**
##
Review of the code written during the microverse admission trials, check if there is need for improvement and if the project runs properly.

## Errors
Errors and bugs should be corrected.

## Repo
Repository should be documented professionally.

## Linters and GitHub Flow
Check if there is no linter error and ensure correct GitHub flow.

## HTML and CSS
Use semantic HTML tags, apply, CSS selectors correctly and box model.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
-# TributePage
\ No newline at end of file
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```
Comment (OyePriscilla) [inline README.md:5]: ## Kindly add descriptive README file to your project and follow [this template](https://github.com/microverseinc/readme-template )

File: index.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    <ui>
+      <li><a href="About-Us">About Us</a></li>
+      <li><a href="Career">Career</a></li>
+      <li><a href="Location">Location</a></li>
+    </ui>
+  </nav>
+<div>
+  <section>
+    <h1 id="welcome-section">Priscilla and Bethelem's PortFolio</h1>
+  </section>
```
Comment (OyePriscilla) [inline index.html:23]: ## The text in the h1 tag is not displayed.
The background-color of the text is the same with the color of the text, change the color of the text or that of the background.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+</head>
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+  <!-- <h1>Hello World! We are software developers</h1>  -->
+  <nav id="nav-bar" class="header">
+    <ui>
+      <li><a href="About-Us">About Us</a></li>
+      <li><a href="Career">Career</a></li>
+      <li><a href="Location">Location</a></li>
```
Comment (OyePriscilla) [inline index.html:17]: ## Hover effects
Kindly apply hover effects to the link as stated in the instructions.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+#welcome-section{
+   display: flex;
+   flex-direction: column;
+   justify-content: center;
+   align-items: center;
+   width: 100%;
+   height: 100vc;
+   background-color: #000;
+
+}
```
Comment (OyePriscilla) [inline style.css:37]: ## Class selector
Class attribute is encouraged to be used not ID attribute.

**Review by OyePriscilla — COMMENTED** [review]
## Good Job so far 

- [ ] HTML semantic tags were used appropriately.
- [ ] GitHub flow observed.
 

## Few issues correct 

- [ ] There is linter error.
- [ ] Adjust the display of your website to look professional; take note of the color as well.
- [ ] Kindly finish up with the remaining sections as required in order to pass all the test cases.

**Review by KayLemba — APPROVED** [review]
Hi @OyePriscilla  😃, 

You did a great job delivering such an amazing review 💯 Great summary 👍 Well done using the GitHub inline comments feature 👍 Polite and well-explained comments 👍

Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**